ceaos/objects/beds.py

```diff
@@ -32,8 +32,8 @@ def delete_sensor(self, name):
         del self.sensors[name]
         self.num_sensors -= 1
 
-    def add_actuators(self, name, actuator):
-        self.actuators[name] = actuator
+    def add_actuators(self, name, new_actuator):
+        self.actuators[name] = new_actuator
         self.num_actuators += 1
```

Comment:
I would drop this num pattern, in python, you can always call `len(actuators)` and get the number of actuators inside.